src/library_browser.js

```diff
@@ -292,7 +292,8 @@ var LibraryBrowser = {
           ctx = GL.getContext(contextHandle).GLctx;
         }
         // Set the background of the WebGL canvas to black
-        canvas.style.backgroundColor = "black";
+        if (!contextAttributes["alpha"])
+          canvas.style.backgroundColor = "black";
```

Comment:
That line influences only when alpha is switched on. Before it was always off for webgl and the background was always invisible because canvas was configured like RGB. I can guess it shouldn't break the tests because they don't use alpha channel in a canvas and no extra changes in SDL and GLUT.